eng/pipelines/templates/build-official-release.yml

```diff
@@ -1,60 +1,135 @@
 # Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
-
+parameters:
+#  - name: SignType (parameter not currently used due to #7935)
+#    type: string
+  - name: SkipOptimize
```

Comment:
Remove

Reply:
As previously commented, that should not be removed as it is a holdover from existing code, and will eventually be used.